airflow/models/skipmixin.py

```diff
@@ -140,6 +140,8 @@ def skip_all_except(self, ti: TaskInstance, branch_task_ids: Union[str, Iterable
         self.log.info("Following branch %s", branch_task_ids)
         if isinstance(branch_task_ids, str):
             branch_task_ids = {branch_task_ids}
+        elif branch_task_ids is None:
+            branch_task_ids = {}
```

Comment:
```suggestion
            branch_task_ids = ()
```

Creating a dict is needless.